package.json

```diff
@@ -92,7 +92,7 @@
     "@bazel/typescript": "^2.2.0",
     "@firebase/app-types": "^0.6.1",
     "@octokit/rest": "18.0.12",
-    "@schematics/angular": "^11.1.0-next.2",
+    "@schematics/angular": "^11.1.0-next.4",
```

Comment:
This should also be `11.1.0-rc.0` correct?

Reply:
yeah just wanted to start the PR early to check for potential CI issues, updated to rc.0 which is available now